### PR TITLE
Wrap phone numbers with parentheses around the area code

### PR DESCRIPTION
**Pull request by mpelzsherman**
Resolves https://github.com/department-of-veterans-affairs/va.gov-team/issues/27501

### CODE REVIEW

**Review by Rleahy22 — APPROVED** [review]
LGTM. Only question is if we want to maintain the parentheses in the display. Good to merge though

**Review by kelsonic — APPROVED** [review]

**Comment by kelsonic** [comment]
Just curious but do we also need to support the TTY: 711 number use case?

File: src/site/filters/liquid.unit.spec.js [hunk 1]
```diff
@@ -1131,6 +1131,22 @@ describe('phoneLinks', () => {
     expect(liquid.filters.phoneLinks(text)).to.equal(expected);
   });
 
+  it('wraps phone numbers with parentheses around the area code', () => {
```
Comment (gescobardev) [inline src/site/filters/liquid.unit.spec.js:1134]: Can we add another test where the phone number looks like this - `(123) 456-7890` ?

**Review by gescobardev — CHANGES_REQUESTED** [review]
Looks great! I just have one comment about testing.

File: src/site/filters/liquid.unit.spec.js (hunk 1 above)
Comment (mpelzsherman) [reply src/site/filters/liquid.unit.spec.js:1134]: Good catch, thanks! Adding.

**Review by gescobardev — APPROVED** [review]

**Pull request merged**